4-toolchain/Dockerfile

```diff
@@ -47,12 +46,12 @@ WORKDIR ${HOME}
 
 # Use entrypoint so path is correctly adjusted already at the time the command
 # is searching, so something like docker run IMG gcc runs binary from SCL.
-COPY contrib/bin/container-entrypoint /usr/bin/container-entrypoint
+COPY root/opt/app-root/bin/container-entrypoint /usr/bin/container-entrypoint
```

Comment:
The same - why not to add `container-entrypoint` into `root/usr/bin/` and don't copy it separately?

@hhorak Do you know why it is needed in `/usr/bin/` (because `/opt/app-root/bin/` is also in PATH) ? Is it necessary to have `container-entrypoint` in both locations now? (same about `usage` script)